_config.yml

```diff
@@ -21,30 +20,6 @@ FLINK_GITHUB_URL: https://github.com/apache/flink
 FLINK_CONTRIBUTORS_URL: https://cwiki.apache.org/confluence/display/FLINK/List+of+contributors
 FLINK_GITHUB_REPO_NAME: flink
 
-optional_components:
-  -
-    name: "Avro"
-    category: "SQL Formats"
-    scala_dependent: false
-    versions:
-      -
-        version: "1.7.2"
-        id: 172-sql-format-avro
-        url: https://repository.apache.org/content/groups/public/org/apache/flink/flink-avro/1.7.2/flink-avro-1.7.2.jar
-        asc_url: https://repository.apache.org/content/groups/public/org/apache/flink/flink-avro/1.7.2/flink-avro-1.7.2.jar.asc
-        sha_url: https://repository.apache.org/content/groups/public/org/apache/flink/flink-avro/1.7.2/flink-avro-1.7.2.jar.sha1
-  -
-    name: "Json"
-    category: "SQL Formats"
-    scala_dependent: false
-    versions:
-      -
-        version: "1.7.2"
-        id: 172-sql-format-json
-        url: https://repository.apache.org/content/groups/public/org/apache/flink/flink-json/1.7.2/flink-json-1.7.2.jar
-        asc_url: https://repository.apache.org/content/groups/public/org/apache/flink/flink-json/1.7.2/flink-json-1.7.2.jar.asc
-        sha_url: https://repository.apache.org/content/groups/public/org/apache/flink/flink-json/1.7.2/flink-json-1.7.2.jar.sha1
-
 # Example for scala dependent component:
```

Comment:
This example is now out-dated.

Reply:
Should I just remove the example? We currently don't have any scala dependent optional components.

Reply:
I'm fine with removing it since the alternative binaries section can serve as a guideline.